### PR TITLE
BEE-36818 [DOCS] Remove `Persistence.ExistingClaim` from migration guides

### PR DESCRIPTION
**Pull request by ammanders**
Removed references to `Persistence.ExistingClaim` from the migration guides.

### CODE REVIEW

**Comment by ammanders** [comment]
@rkivisto I only removed the lines in the `migration-values-example.yaml` example that reference `Persistence.ExistingClaim` for now, unless you think the entire example can now be removed. Thank you!

File: helm-custom-value-file-examples/migration-values-example.yaml [hunk 1]
```diff
@@ -15,6 +15,4 @@ OperationsCenter:
       SecretName: core-example-com-tls
       Host: jenkins.cluster.local
```
Comment (rkivisto) [inline helm-custom-value-file-examples/migration-values-example.yaml:17]:
```suggestion
```

File: helm-custom-value-file-examples/migration-values-example.yaml [hunk 2]
```diff
@@ -15,6 +15,4 @@ OperationsCenter:
...
 
-# Do not change these values. They are all set for migrating CloudBees CI. 
-Persistence:
-  ExistingClaim: 'jenkins-home-cjoc-0' 
+
```
Comment (rkivisto) [inline helm-custom-value-file-examples/migration-values-example.yaml:18]:
```suggestion
```

File: helm-custom-value-file-examples/openshift-migration-values-example.yaml [hunk 3]
```diff
@@ -19,7 +19,4 @@ OperationsCenter:
     Project:
       name: myproject
```
Comment (rkivisto) [inline helm-custom-value-file-examples/openshift-migration-values-example.yaml:21]:
```suggestion
```

File: helm-custom-value-file-examples/openshift-migration-values-example.yaml [hunk 4]
```diff
@@ -19,7 +19,4 @@ OperationsCenter:
...
 
-# Do not change these values. They are all set for migrating CloudBees Core.
-Persistence:
-  ExistingClaim: 'jenkins-home-cjoc-0'
```
Comment (rkivisto) [inline helm-custom-value-file-examples/openshift-migration-values-example.yaml:22]:
```suggestion
```

**Review by rkivisto — APPROVED** [review]
Looks good, I've also confirmed the current chart does not include the value:

```
$ helm show values cloudbees/cloudbees-core|grep ExistingClaim
$ helm search repo cloudbees/cloudbees-core
NAME                    	CHART VERSION         	APP VERSION	DESCRIPTION                                   
cloudbees/cloudbees-core	3.19005.0+21d3f150a75f	2.462.2.2  	Enterprise Continuous Integration with Jenkins
```

It's minor, but the blank lines can also be removed.

**Review by kcsmalls — APPROVED** [review]

**Pull request merged**